environments/values-idfdemo.yaml

```diff
@@ -0,0 +1,23 @@
+name: "idfdemo"
+fqdn: "demo.lsst.cloud"
+appOfAppsName: "science-platform"
+butlerRepositoryIndex: "s3://butler-us-central1-repo-locations/data-dev-repos.yaml"
+butlerServerRepositories:
+  dp02: "https://demo.lsst.cloud/api/butler/repo/dp02/butler.yaml"
+gcp:
+  projectId: "science-platform-demo-9e05"
+  region: "us-central1"
+  clusterName: "science-platform-demo"
+onepassword:
+  connectUrl: "https://roundtable-dev.lsst.cloud/1password"
```

Comment:
Shouldn't this use production? No need to have exposure to a test cluster.

Reply:
My reasoning is that this cluster is closer to idf-dev than it is to anything else.  But I also don't really care.  Prod's fine.